### PR TITLE
feat: Migrate to Casbin.NET 2.0.0-preview

### PR DESCRIPTION
**Pull request by sociometry**
Feat: Migrate Casbin.NET dependency from 1.x to 2.0.0-preview

### CODE REVIEW

File: Casbin.Adapter.EFCore.UnitTest/Fixtures/ModelProvideFixture.cs [hunk 1]
```diff
@@ -1,15 +1,15 @@
-﻿using System.IO;
-using NetCasbin.Model;
+﻿// using System.IO;
```
Comment (sagilio) [inline Casbin.Adapter.EFCore.UnitTest/Fixtures/ModelProvideFixture.cs:1]: Why comment about this?

**Review by sagilio — APPROVED** [review]
LGTM

File: Casbin.Adapter.EFCore.UnitTest/Fixtures/ModelProvideFixture.cs (hunk 1 above)
Comment (sociometry) [reply Casbin.Adapter.EFCore.UnitTest/Fixtures/ModelProvideFixture.cs:1]: `File` will be recognized as `Casbin.Adapter.File`, so I used `System.IO.File`. Thus `using System.IO;` is useless.

Comment (sagilio) [reply Casbin.Adapter.EFCore.UnitTest/Fixtures/ModelProvideFixture.cs:1]: ok, I see, you can remove it directly.

Comment (sociometry) [reply Casbin.Adapter.EFCore.UnitTest/Fixtures/ModelProvideFixture.cs:1]: Thanks, I'll keep this in mind.

**Pull request merged**